os/arch/arm/src/amebad/amebad_spi.c

```diff
@@ -266,10 +269,11 @@ static struct amebad_spidev_s g_spi1dev = {
 	.spi_object = {0},
 
 	.spi_idx = MBED_SPI1,
-	.spi_mosi = PB_4,
-	.spi_miso = PB_5,
-	.spi_sclk = PB_6,
-	.spi_cs = PB_7,
+	.spi_mosi = PA_12,
```

Comment:
We should keep previous value, because we will not divide cs to cs0 & cs1 anymore. and please check pinmap what I shared with eric kim

Reply:
I will revert spi1 back to it's previous pin map